File: Java_Backend/E-Tour/src/main/java/dto/PassengerDTO.java
package dto;

import lombok.*;
import java.time.LocalDate;
import entity.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class PassengerDTO {
    private Integer paxId;
    private String paxName;
    private LocalDate paxBirthdate;
    private Integer paxTypeId;     
    private String paxTypeName;     
    private Double paxAmount;
    private Integer bookingId;
    private PaxType paxType;
}

File: DotNet_Backend/SafarnamaApplication/SafarnamaApplication/DTOs/PassengerDTO.cs
using System.Diagnostics.Eventing.Reader;
using SafarnamaApplication.Models;

namespace SafarnamaApplication.DTOs
{
    public class PassengerDTO
    {
        public int PaxId { get; set; }
        public string? PaxName { get; set; }
        public DateOnly? PaxBirthdate { get; set; }
        public int? PaxTypeId { get; set; }
        public string? PaxTypeName { get; set; }
        public double? PaxAmount { get; set; }
        public int? BookingId { get; set; }

        public PaxType? PaxType { get; set; }
    }
}
